package.json

```diff
@@ -77,7 +77,7 @@
     "postmerge": "npm install"
   },
   "bootstrapKickstart": {
-    "bundleCSS": {},
+    "bundleCSS": [],
```

Comment:
Please check if bundling works this way.
Test with the following for example:
https://github.com/micromata/bootstrap-kickstart/commit/015b25d99d03b61a572c7de26c6a6de5f61cc7f4#diff-b9cfc7f2cdf78a7f4b91a753d10865a2L78